source/routing/raptor_api.cpp

```diff
@@ -675,7 +674,6 @@ make_pathes(PbCreator& pb_creator,
     pb_creator.set_response_type(pbnavitia::ITINERARY_FOUND);
     add_pathes(pb_creator, paths, worker, direct_path,
                origin, destination, datetimes, clockwise);
-
     if (pb_creator.empty_journeys()) {
         pb_creator.fill_pb_error(pbnavitia::Error::no_solution,
```

Comment:
So you have to keep it here?
You couldn't update the tests so they match without that?